### DISCUSSION TITLE
is there a default user resource?

### DISCUSSION THREAD

**Discussion by jdion84** [filamentphp/filament#7760] [Help] [answered]
is there a default user resource like nova has, or do I need to create one from scratch myself?

other than that, loving filament so far.

**Comment by bezhanSalleh** [comment]
No there isn't. Best way to get started with filament is to just pull-in the demo repo, most of the things you would need is already implemented there. and of course you can easily copy/pase the user resource from there as well.

**Comment by danharrin** [comment, marked as answer]
`php artisan make:filament-resource User --generate` will do most of it for you

**Reply by jdion84** [reply]
Thanks to you both.
